src/index.ts

```diff
@@ -41,19 +41,22 @@ async function getRealNames(repoList: ReturnType<typeof getAllRepoDeps>, config:
 				.then(content => {
 					const name = content?.name || ""
 					const version = content?.version || ""
-					return { name: name,
-						 version: version
-						}
+					const languageVersion = content?._nodeVersion || undefined
```

Comment:
Why `undefined`, why not `null`?

Reply:
Also down the road, if we want to store the value in JSON, then it won't work because JSON doesn't accept `undefined`

Reply:
Undefined mirrors the return value when a field isn't present at all. When we convert it to json (in depDataToJson), that field isn't present at all if it's undefined.

Reply:
Ok sounds good, could you document it in the readme, so that its clear what the the crawler returns